Gems/Atom/RPI/Code/Source/RPI.Builders/Material/MaterialBuilder.cpp

```diff
@@ -547,6 +547,12 @@ namespace AZ
                     return;
                 }
 
+                for (auto& dependency : jobProduct.m_dependencies)
+                {
+                    // Set the load behavior to PreLoad, without clearing any other product dependency flags
+                    Data::ProductDependencyInfo::SetAssetLoadBehavior(dependency.m_flags, Data::AssetLoadBehavior::PreLoad);
+                }
```

Comment:
I'm curious why set the load behavior this way, rather than setting it on the Asset<ImageAsset> reference inside the MaterialAsset class. What's the difference?